Lib/idlelib/help.py

```diff
@@ -191,6 +198,52 @@ def findfont(self, names):
                                   for x in tkfont.families(root=self)):
                 return name
 
+    def configure_tags(self):
+        "Configure tags used in parsing."
+        self['font'] = self.fonts['normal']
+        for tag in ('em', 'h1', 'h2', 'h3', 'pre', 'preblock'):
+            self.tag_configure(tag, font=self.fonts[tag])
+        self.tag_configure('pre', background='#f6f6ff')
+        self.tag_configure('preblock', lmargin1=25, borderwidth=1,
+                           relief='solid', background='#eeffcc')
+        self.tag_configure('l1', lmargin1=25, lmargin2=25)
+        self.tag_configure('l2', lmargin1=50, lmargin2=50)
+        self.tag_configure('l3', lmargin1=75, lmargin2=75)
+        self.tag_configure('l4', lmargin1=100, lmargin2=100)
+
+    def bind_events(self):
+        "Bind events."
+        # Zoom out works with or without shift.
+        self.event_add('<<zoom-text-out>>',
+                       *['<Control-Key-equal>', '<Control-Key-plus>'])
+        self.bind('<<zoom-text-out>>', lambda e: self.zoom(e, 'out'))
+
+        self.event_add('<<zoom-text-in>>', '<Control-Key-minus>')
+        self.bind('<<zoom-text-in>>', lambda e: self.zoom(e, 'in'))
```

Comment:
I originally did this because I wasn't sure about hardcoding the checks for the 'plus' and 'minus' keys, if those would be configurable at some point.  I've changed it to look for the keys and I suppose it can be changed later to use the keys attached to the binding.

Reply:
I think, if anything, we have too many key configurations.  zoom has been changed.  See new review comment.